pkg/operator/options/suite_test.go

```diff
@@ -110,6 +110,8 @@ var _ = Describe("Options", func() {
 				LogErrorOutputPaths:     lo.ToPtr("stderr"),
 				BatchMaxDuration:        lo.ToPtr(10 * time.Second),
 				BatchIdleDuration:       lo.ToPtr(time.Second),
+				PreferencePolicy:        lo.ToPtr(options.PreferencePolicyRespect),
```

Comment:
It looks like we still need to add the check for these fields in `expectOptionsMatch`